multinode/bootstrapping.md

```diff
@@ -0,0 +1,201 @@
+that the `timescaledb` extension needs. The schema name is decided by
+looking at what schema name is used for the extension on the access node and
+using the same name on the data node. If you have created the
+`timescaledb` extension in the default manner, `public` will be used
+for the schema on the access node, hence `public` will be used on the
+data node as well.
+
+If the schema name for the extension on the access node is `public`,
+no new schema is created, but otherwise it is expected that the schema
+name does *not* exist. In the event that the schema name exists (but
+is not `public`), an error will be raised and the bootstrapping will abort.
+
+Alternative description:
+> If the schema already exists on the data node the access node will
+> verify that it is correctly configured and proceed with installing
+> the extension.
+
+To change the schema used on the access node and data node, you can
+create a new schema, then create the extension with the schema that
+you want:
+
+```
+CREATE SCHEMA magic;
+CREATE EXTENSION timescaledb SCHEMA magic;
+```
+
+When validating the schema, it is necessary that the schema is owned
+by the current user, or is `public`.
+
+>:TIP: This will place all the Timescale functions (for example,
+> `create_hypertable` in the `magic` schema), so you need to qualify
+> all function references with `magic`. For example:
+>
+> ```
+> SELECT * FROM magic.create_hypertable('conditions', 'time', 3);
+> ```
+
+### Installing the extension
+
+After the schema is created, the extension is installed using the
+`CREATE EXTENSION` command with the `SCHEMA` option to refer to the
+schema from the section above. The schema name for the extension will
+be either `public` or a dedicated schema name.
+
+If the extension already exists in the database, an error will be
+raised and the bootstrapping aborted.
+
+Proposed changes:
+> If the extension already exists, it is validated to ensure that it
+> can be used with the access node and the procedure proceeds to set
+> the metadata.
+
+### Validating the extension
+
+Validation of the extension checks that the extension on the data node
+is of a version that is compatible with the version on the access node
+and that the access node has permissions to use the extension.
+
+The extension on the data node is compatible with the extension on the
+access node if the major versions are the same and the minor version
+on the data node is before or the same as the minor version on the access
+node. For example:
+
+- Compatible: data node v1.3.1, access node v1.5.1
+- Compatible: data node v1.3.5, access node v1.3.1
+- Incompatible: data node 1.4.1, access node v1.3.1
+- Incompatible: data node 2.0.0, access node v1.3.1
+
+In addition to validating the version, it is necessary to ensure that
+the access node has sufficient privileges on the data node. This is
+checked by ensuring that the connected user is the owner of the
+extension.
+
+## Set the metadata
```

Comment:
This seems very much like implementation detail. Do we need this?

Reply:
It is a very small detail so I do not see it as a big problem to add. Also, we cannot really change this between versions, so it is very stable.

For need: there are DBAs that need to repair or understand a problem. This is really where the documentation is really needed.

Reply:
Ok, I buy that.